test-frame-log-collector/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.skodjob</groupId>
+        <artifactId>test-frame</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>test-frame-log-collector</artifactId>
+
```

Comment:
you miss complete required stuff for releasing like there -> https://github.com/skodjob/test-frame/blob/eaf1c556e17b7024eb88a080b816b5cfa199d7b9/test-frame-common/pom.xml#L15 like license, name, scm etc...

Reply:
good point, added